### DISCUSSION TITLE
Why don't you have the Source-Code in GitHub?

### DISCUSSION THREAD

**Discussion by schittli** [NexovaDev/UpdateHub#300] [Q&A]
Hello

I just wanted to support the project and answer the question _[Does UpdateHub collect any Data? (GDPR / DSGVO)](https://github.com/KK-Designs/UpdateHub/discussions/293)_ and do the code review 😃, so that I could afterwards also contribute.

Unfortunately, the source code is not on this GitHub Repository. So my question is: 
* Why are you hiding the source code? 
* Is there a way to do a code review?

Thanks a lot, kind regards,
Thomas

**Comment by onyx-nxt** [comment]
Hi,
We have not decided whether to open source and keep the product free or to commercialize it. We will do so based on the usage trajectory of UpdateHub. We intend to keep it free for non-commercial users. It also takes effort and resources to maintain the app so we might monetize it for commercial users or adopt a freemium model. For each release, I provide a [VirusTotal scan](https://www.virustotal.com/gui/file/80a081b63fac71e96930c24c342ed51a184e9bca964633d7d898b07b82f90b92/ "VirusTotal scan"). Please feel free to review the virus scan result. \
\
Hope this answers your questions.
